tests/baselines/reference/generatorTypeCheck13.types

```diff
@@ -3,7 +3,7 @@ function* g(): IterableIterator<number> {
 >g : () => IterableIterator<number>
 
     yield 0;
->yield 0 : any
+>yield 0 : undefined

```

Comment:
that's strange, I thought `yield 0` was `any` in other tests

Reply:
The result type for `yield 0` is derived from the *iterator next type* of `IterableIterator<number>`, which is `undefined`.